parsers/BO.py

```diff
@@ -65,7 +65,10 @@ def fetch_hourly_production(country_code, obj, hour, date):
         hour = 24
     # Fill production types
     for i_type in MAP_GENERATION.keys():
-        data['production'][i_type] = obj[MAP_GENERATION[i_type]][obj.hour == hour].iloc[0]
+        try:
+            data['production'][i_type] = obj[MAP_GENERATION[i_type]][obj.hour == hour].iloc[0]
+        except KeyError as e:
+            data['production'][i_type] = None
```

Comment:
Ideally I think you want to drop the whole hour when wind data is not available (as this datapoint is not "consistent").
Maybe the best way is to abort the method and return `None` in that case, filtering out missing hours later on?

Reply:
Ok I can do that instead.